React/my-app/src/screens/home/App.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import data from "./data.json";
+import "./App.css";
+import defaultImg from "./photos/default.png";
+import { WBOOKS, SELECCIONAR_FILTRO, TITULO, AUTOR } from "./strings.js";
+
+class App extends Component {
+  state = { books: data, searchType: null };
+
+  render() {
+    return (
+      <div className="mainContainer">
+        <h1 className="main_title">{WBOOKS} </h1>
+        <form>
+          <select
+            className="selector"
+            name="select"
+            onChange={this.setSearchType}
+          >
+            <option value="null">{SELECCIONAR_FILTRO}</option>
+            <option value="title">{TITULO}</option>
+            <option value="author">{AUTOR}</option>
+          </select>
+          <input
+            className="search"
+            type="text"
+            placeholder="Buscar..."
+            name="fname"
+            onChange={this.filterBooks}
+          />
+        </form>
+        <div className="gallery">
+          {this.state.books.map(element => <Book book={element} />)}
+        </div>
+      </div>
+    );
+  }
+
+  setSearchType = search => {
```

Comment:
functions should be placed on top of class.
first all functions, then render function.

state
functions of class
render function